lib/src/common/model/checks.dart

```diff
@@ -94,7 +94,8 @@ class CheckRun {
   final int? checkSuiteId;
   final String? detailsUrl;
   final DateTime startedAt;
-  final CheckRunConclusion conclusion;
+  final CheckRunConclusion? conclusion;
+  final String? slug;
```

Comment:
Hi Casey, I previously considered using RepositorySlug but finally decided to use String instead because I discovered that the format of checkRun's app-slug here doesn't really fit the format of RepositorySlug.

Reply:
Hi Casey, I checked in the check runs payload here https://api.github.com/repos/flutter/packages/commits/68d068dfda0cf5580290c278bfa3f3b7ff657654/check-runs, I think the app.slug of checkRun is not the same thing as the repositorySlug of its parent pull request.
That means, even we query for the checkRuns from one pull request, each check run's app.slug is different from each other, and also not same as its parent pull request's slug.